data_loader.py

```
@@ -0,0 +1,73 @@
+import numpy as np
+
+class MotorImageryDataset:
+    def __init__(self, data_path, subject, session='T'):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)